### DISCUSSION TITLE
Sabnzbd Container

### DISCUSSION THREAD

**Discussion by StabbingHobo** [tteck/Proxmox#977] [Help Needed] [answered]
Just installed the Sabnzbd lxc container -- install was fine.  However; it doesn't appear to be extracting an archive after a completed download.

Navigating to the folder shows all the archives, but nothing extracted.  I've changed no settings that would be related to extraction, just sorting and post processing flags.

In Config > Special I have enable_unzip/unrar/7zip all selected as well.

Any suggestions would be great.

**Comment by tteck** [comment]
Could be because 7zip isn't installed by the script. **Adding it now**. You can install it to see if that fixes your issue
 `apt-get install p7zip-full`

**Comment by StabbingHobo** [comment]
Error I'm just noticing now is:

 _Your UNRAR version is 0.00, we recommend version 5.50 or higher_

Edit:  This is following installing p7zip as per your suggestion.
Edit #2: Unrar --version shows 0.0.1

**Reply by tteck** [reply]
The only other thing that I can think of is

```
apt autoremove --purge unrar-free
wget -q http://http.us.debian.org/debian/pool/non-free/u/unrar-nonfree/unrar_6.0.3-1+deb11u1_amd64.deb
dpkg -i unrar_6.0.3-1+deb11u1_amd64.deb
rm unrar_6.0.3-1+deb11u1_amd64.deb
```

**Reply by StabbingHobo** [reply]
In my haste I blew away the container last night to start from scratch.  Instead, I went with the manual installation option via CLI.

So, using **your** script for Plex LXC which I had already installed, I followed the steps [here](https://www.linuxbabe.com/ubuntu/install-sabnzbd-ubuntu-16-04) to get the app running and tested it working.

The install line:
`sudo apt install sabnzbdplus python-sabyenc par2-tbb`

I did not use, just opting to install sabnzbdplus alone.  I also opted to make one other change in that guide to have Sabnzbd use port 8989.

Hope some of this information is useful.

**Reply by tteck** [reply]
The only bad thing is, using apt packages are usually older versions.

### example:
Using Debian, `apt install sabnzbdplus` **SABnzbd Version 3.1.1** [released Nov 11, 2020](https://github.com/sabnzbd/sabnzbd/releases/tag/3.1.1)

While the script downloads directly from their repo **SABnzbd Version 3.7.1** [released Dec 17, 2022](https://github.com/sabnzbd/sabnzbd/releases/tag/3.7.1)

Ubuntu may have newer releases?

**Reply by StabbingHobo** [reply]
I didn't manually perform an update following the installation -- however; in saying that, Sab shows an installed version of 3.7.1

I wanted to make that distinction as the most recent version appears to be installed following the apt install.  The Plex LXC container does install Ubuntu 20.04.5 LTS (of course, you already knew that ;) )

**Reply by tteck** [reply]
Cool, I'll try Ubuntu. So, did you have to install unrar, unzip ect...?

**Reply by StabbingHobo** [reply]
I can't say that for certain to be honest.

Using a Plex LXC container that was installed using your script and already in place, I simply followed the aforementioned guide.  Actually; now that I'm reviewing the same guide I posted earlier -- I cannot honestly remember if I added the repos listed or not.  Guess that fundamentally answers the repo statement from before:

```
sudo add-apt-repository ppa:jcfp/nobetas

sudo add-apt-repository ppa:jcfp/sab-addons

sudo apt update

sudo apt install sabnzbdplus python-sabyenc par2-tbb
```

**Reply by StabbingHobo** [reply]
Okay; I walked through it again -- I did not previously add the repos listed.  I just added them and was able to install the par2-tbb package as well.  (Not important, just worth mentioning)

**Reply by tteck** [reply]
Strange, using the Ubuntu LXC (22.04), `apt install sabnzbdplus` installs SABnzbd Version 3.5.1 better than Debian
It does automatically install p7zip-full

I'm going to stick with the up to date repo releases and try using the `unrar-nonfree.deb`

**Reply by tteck** [reply]
BTW, besides 7zip, it also installed `UNRAR 6.11 beta 1 freeware`

**Reply by tteck** [reply]
If you get a chance, I'd be grateful if you'd try the script again with the changes.

**Reply by StabbingHobo** [reply]
Sure; I'll give it a go in a little bit and run it through it's paces.

**Reply by StabbingHobo** [reply]
Installed using default settings -- but because I'm lazy -- I restored my 'backup' from my current working install (just settings/server connection info) and performed a download.

[Worked](https://imgur.com/a/6Dcjgxu)

The only thing I'd say is, perhaps a note on your page indicating that a user may want to define a greater storage space availability.  After install there is only 3.8G remaining on the drive, all the temp files download locally which would consume that space very fast.  Even when the incomplete file location is defined elsewhere oddly....

[Space](https://imgur.com/a/aSGNTrN)

**Reply by tteck** [reply]
Thank you for helping me get this resolved!!

**Reply by StabbingHobo** [reply]
> Thank you for helping me get this resolved!!

NO!

Thank you for all your hard work.  I've been using your scripts since day one of ProxMox :)
Glad I could contribute a minutia of help!

**Comment by tteck** [comment, marked as answer]
Script has been updated to resolve issue
